renovate.json

```diff
@@ -1,31 +1,13 @@
 {
-  "extends": [
-    "config:base",
-    ":disableDependencyDashboard"
-  ],
-  "assigneesFromCodeOwners": true,
-  "timezone": "Europe/London",
-  "schedule": [
-    "after 9am every weekday",
-    "before 5pm every weekday"
-  ],
```

Comment:
We now inherit this from https://github.com/ministryofjustice/hmpps-renovate-config/blob/4acc8951af908562c507a76b68ead835193c6a31/base.json#L8-L12